iGCS/Managers/GCSSpeechManager.m

```diff
@@ -31,21 +33,35 @@ - (instancetype)init {
         // set speech defaults
         _utteranceRate = 0.1f;
         _defaultLanguage = @"en-US";
+        _notificationCenter = [NSNotificationCenter defaultCenter];
+        _speakerQueue = [[NSOperationQueue alloc] init];
 
+        _speechSynthesizer = [[AVSpeechSynthesizer alloc] init];
+
+        __weak GCSSpeechManager *weakSelf = self;
```

Comment:
For my own education, what does this achieve for us? Is this just a convention?

Understand that this will prevent the GCSSpeechManager from being kept live by the closures passed to the notification centre, but not clear why that matters? (as the speech manager is a singleton retained for the entire application)